pkg/fileservice/mem_cache.go

```diff
@@ -49,8 +48,7 @@ func (m *MemCache) Read(
 ) (
 	err error,
 ) {
-	_, span := trace.Start(ctx, "MemCache.Read")
-	defer span.End()
+	metric.FsMemCacheReadCounter.Inc()
```

Comment:
as i concerned, MemCacheCount should have 2 type: `hit` & `total`  or  `hit` & `miss`.

Reply:
Sure. I will add the Hit & Miss rates as well. I will refactor this code to be more flexible.

Reply:
This PR should address the same: https://github.com/matrixorigin/matrixone/pull/8351

PS: Miss is not explicit in MemCache. We are now exposing NumRead & NumHit. NumMiss is basically NumRead-NumHit.